pio/lib/Sender/Sender.cpp

```diff
@@ -525,6 +525,75 @@ bool SenderClass::sendInfluxDB(String server, uint16_t port, String db, String n
   return true;
 }
 
+bool SenderClass::sendInfluxDBHTTPS(String server, uint16_t port, String db, String name, String username, String password)
+{
+  std::unique_ptr<BearSSL::WiFiClientSecure> client(new BearSSL::WiFiClientSecure);
+  client->setInsecure();
+
```

Comment:
Looks like SenderClass::sendInfluxDBHTTPS differs from SenderClass::sendInfluxDB by only lines 530 and 531 (and line 540 where you refer to client). 
Was it worth to copypaste the rest of the SenderClass::sendInfluxDB implementation?
May be simply to introduce a boolean flag bUseHTTPS to SenderClass::sendInfluxDB arguments and either create new local client or use existing global one ?

Reply:
Reimplemented as a bool as suggested. There is still some slight duplication due to differing function names, but I tried to keep it to a minimum.